### PR TITLE
feature(product list component): Fetched Products from the database 

### PR DESCRIPTION
**Pull request by 27ManishaPatel**
# Description
Fetched Products from the database  (images still from /assets)
Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?
http://localhost:3000/collections
Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/ProductList/ProductList.component.css [hunk 1]
```diff
@@ -24,17 +25,17 @@
...
-}
-.product p {
+} */
+/* .product p {
   color: black;
   text-align: left;
   padding: 10px;
   font-family: Montserrat;
-}
+} */
```
Comment (shpomp) [inline packages/client/src/components/ProductList/ProductList.component.css:38]: 🗑️

File: packages/client/src/components/ProductList/ProductList.component.js [hunk 2]
```diff
@@ -1,21 +1,45 @@
...
   return (
-    <div className="product-list-container">
-      <div className="rendered-product">{ListOfProducts}</div>
+    <div>
+      <div className="product-list-container">
+        {isLoading && <p>Loading...</p>}
+        {products.length === 0 ? (
+          <p>There is no product available</p>
+        ) : (
+          <div className="rendered-product">{ListOfProducts}</div>
```
Comment (shpomp) [inline packages/client/src/components/ProductList/ProductList.component.js:40]: Nice!
however, this logic makes the "loading" and "There is no product available" always show at the same time - until they are fetched. 
Make a Jira and take care of this some time later 🙏🏽

**Review by shpomp — APPROVED** [review]
Very nice! 👏🏽

File: packages/client/src/components/ProductList/ProductList.component.js (hunk 2 above)
Comment (27ManishaPatel) [reply packages/client/src/components/ProductList/ProductList.component.js:40]: Ohh yes sure will do that in next task thanks!

**Review by patel-prakashkumar — APPROVED** [review]

**Pull request merged**